crates/cxx-qt-lib/src/gui/qpainterpath.cpp

```diff
@@ -0,0 +1,26 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+// SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+#ifdef CXX_QT_GUI_FEATURE
```

Comment:
Will need to change to stay in sync with the removal of `CXX_QT_GUI_FEATURE` for cpp files